src/main/java/app/model/User.java

```diff
@@ -0,0 +1,95 @@
+package app.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+@SQLDelete(sql = "UPDATE user SET is_deleted = true WHERE id=?")
+@SQLRestriction(value = "is_deleted = false")
```

Comment:
You can use the @SoftDelete(columnName = ‘is_deleted’) annotation instead of @SQLDelete and @SQLRestriction. After that, you can easily remove the isDeleted field from the User class.

Reply:
will the code work if one of the fields is like this:
![image](https://github.com/jv-feb24-team04/task-management-app/assets/159646520/2c13e951-1974-47bc-acf5-b9187d8bd773)

Reply:
Hmm, that's a very interesting question, I hadn't even thought about it. I'll have to test the behavior.